packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -89,37 +100,16 @@ export class RemoteTableService {
       throw new NotFoundException('Remote server does not exist');
     }
 
-    const dataSourcesMetatada =
```

Comment:
still not a fan of this updateRemoteTableSyncStatus function. it's not clear when we call it that is has a lot of effects (deleting tables), and inside it's no much more than a switch. I would have two functions: sync + unsync that would do the effects

Reply:
Alright I will separate endpoints in my next PR that is supposed to connect this endpoint with the front 👍